views/sideVideoView.js

```diff
@@ -0,0 +1,11 @@
+var SideVideoView = Backbone.View.extend({
+  initialize: function(model){
```

Comment:
You shouldn't need to write this function, backbone already does this if you pass a model